src/Corrade/Containers/Test/PairStlTest.cpp

```diff
@@ -83,6 +85,46 @@ void PairStlTest::convertMove() {
     CORRADE_VERIFY(!c.second);
 }
 
+void PairStlTest::structuredBindings()
+{
+#if CORRADE_CXX_STANDARD >= 201703
```

Comment:
This way it won't get tested on *any* CI jobs, and neither for anyone who doesn't explicit explicitly pass `-std=c++17` to the whole project. Which people rarely do.

It needs an explicit new test file, as I mentioned in #127 (and #157).